docs/source/databases.rst

```diff
@@ -189,3 +191,85 @@ writing, and would always bind to a master database with read/write
 privileges).  It's also possible to extend
 :class:`~pecan.hooks.TransactionHook` or write your own hook implementation for
 more refined control over where and when database bindings are called.
+
+Assuming a master/standby setup, where the master accepts write requests and
+the standby can only get read requests, a Pecan configuration for sqlalchemy
+could be::
+
+    # Server Specific Configurations
+    server = {
+        ...
+    }
+
+    # Pecan Application Configurations
+    app = {
+        ...
+    }
+
+    # Master database
+    sqlalchemy_w = {
+        'url': 'postgresql+psycopg2://root:@10.0.0.1/dbname',
+        'echo':          False,
```

Comment:
The examples at the beginning where using it, so I was trying to keep things consistent. But sure, I will update this.